README.md

```diff
@@ -367,6 +367,7 @@ Most configuration fields are named such that they can be defaulted to falsey. A
 | eventsLimitInMem | number | 10000 | [Optional] The number of events that can be kept in memory before the SDK starts to drop events when not using Session Storage (the default).
 | disableIkeyDeprecationMessage | boolean | true | [Optional]  Disable instrumentation Key deprecation error message. If true, error message will NOT be sent. **Note: instrumentation key support will end soon**, see aka.ms/IkeyMigrate for more details.
 | bufferOverride <br/><sub>since 2.8.12</sub> | IStorageBuffer | undefined | [Optional] Identifies a simple interface to allow you to override the storage mechanism used for tracking unsent and unacknowledged events, when not provided defaults to using SessionStorage interface. You MUST supply both the `getItem` and `setItem` functions when defined.
+| storagePrefix | string[] | undefined | [Optional] An optional value that will be added as name prefix for storage name. |
```

Comment:
Also add a "since" as this doesn't exist in the eariler versions.

Probably add some wording about why, looks at the issue or maybe just link to the issue might work.